lib/streamlit/elements/dataframe_selector.py

```diff
@@ -25,7 +25,7 @@
 from streamlit.runtime.metrics_util import gather_metrics
 
 if TYPE_CHECKING:
-    from altair.vegalite.v4.api import Chart
+    from altair import Chart
```

Comment:
Does this "from altair import Chart" import work for Altair 4?

Reply:
Yep, `from altair import Chart` is the actual public API that always existed this way :)